README.md

```diff
@@ -64,28 +68,20 @@ You will need the following groups of dependencies to run the project successful
    - [SQL Server 2017+](https://www.microsoft.com/en-gb/sql-server/sql-server-downloads)
    - [Docker and Docker Compose](https://docs.docker.com/)
 
-3. To emulate azure storage you will require:
-   - [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator)
+3. To emulate Azure storage services (blobs, tables and queues) you will require:
 
-  The installer for Windows x64 is 
-  [here](https://download.microsoft.com/download/3/9/F/39F968FA-DEBB-4960-8F9E-0E7BB3035959/SQLEXPR_x64_ENU.exe).
-  
-  You can then run Azure Storage Emulator using the default instance of SQL Server as its data source, with the command:
-  
-  ```
-  AzureStorageEmulator.exe init /server .
-  ```
-  
-   You will currently need to use Windows (or a Windows VM with ports 10001, 10002 and 10003 exposed) to run the service 
-   due to a dependency on Azure Storage Emulator for development, specifically the table storage component which is not 
-   supported by [Azurite](https://github.com/Azure/Azurite). However this does not apply to every component of the 
-   service.
+   - [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator) -
+     installer for Windows x64 is [here](https://download.microsoft.com/download/3/9/F/39F968FA-DEBB-4960-8F9E-0E7BB3035959/SQLEXPR_x64_ENU.exe)
+   - [Docker and Docker Compose](https://docs.docker.com/)
```

Comment:
This sounds like they're both the same option. I would reword it make it clear all the different options you have - azure storage emulator, docker/azurite, VM, etc. It could even be its own section maybe.

Reply:
Good point, will update the wording slightly